README.md

```diff
@@ -1,2 +1,157 @@
+
+	var cesiumSource = path.resolve(__dirname, '../path/to/cesium/Source');
+
+You could even point the cesium to a branch in GitHub or another url in your `package.json` file.
+
+### Source Maps
+
+Enable source maps in development for easier debugging. They are only available when using the development server. There are many [options](https://webpack.js.org/configuration/devtool/). Suggested would be `eval` for fast build and rebuilt time and allowing evaluation of the webpack generated code.
+
+Source maps can be enabled with the following config object:
+
+	devTool: `eval`
+
+### Performance Configurations 
+
+The following optimizations are recommended for building for production and will increase performance and result in smaller bundle sizes.
+
+For best performance, make sure you are requiring individual modules from Cesium instead of the global Cesium object. Additionally, only copy the static assets that your app requires with the `CopyWebpackPlugin` by taking advantage of the [pattern options](https://github.com/webpack-contrib/copy-webpack-plugin#pattern-properties).
+
+**For the numbers, see [bundle size comparisions](#bundle-size-comparisons)**
+
+##### Removing pragmas
+
+To remove pragmas like a traditional cesium release build, use the [`strip-pragma-loader`](https://www.npmjs.com/package/strip-pragma-loader).
+
+Install the plugin with npm,
+
+```
+npm install strip-pragma-loader --save-dev
+```
+
+and include the loader in `module.rules` with `debug` set to `false`.
+
+```
+rules: [{
+	test: /\.js$/,
+	enforce: 'pre',
+	include: path.resolve(__dirname, cesiumSource),
+	use: [{
+		loader: 'strip-pragma-loader',
+		options: {
+		    pragmas: {
+				debug: false
+			}
+		}
+	}]
+}]
+```
+
+##### Uglify and Minify
+
+Compress the final size of the bundle by minifying included JavaScript using UglifyJS with the [`uglifyjs-webpack-plugin`](https://webpack.js.org/plugins/uglifyjs-webpack-plugin/).
+
+Install the plugin,
+
+```
+npm install uglifyjs-webpack-plugin --save-dev
+```
+
+require it,
+
+```
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+```
+
+and include it in the list of plugins.
+
+```
+plugins: [
+	new UglifyJsPlugin()
+]
+```
+
+Additionally, minify the CSS files when loaded with the `css-loader`
+
+```
+module: {
+	rules: [
+	{
+		test: /\.css$/,
+		use: [ 
+			'style-loader', 
+			{
+				loader: 'css-loader',
+				options: {
+					minimize: true
+				}
+			}
+		]
+	},
+}
+```
+
+## Bundle size comparisons
+
+Here is a comparison of the Size of the separated `cesium.js` bundle for `release` and `build` configurations, for using the global `Cesium` object and including individual modules for the Hello World viewer.
+
+|     | `build` | `release` |
+| --- | --- | --- |
+| `Cesium` object | 9.91 MB | 2.82 MB |
+| modules | 7.43 MB | 1.91 MB |
```

Comment:
Also provide gzipped values for these numbers (and label them as such) that's the value that really mattes in the end.